Automattic/jetpack
# Publicize: enabled feature by default

This PR enables the Publicize feature as default, setting the `jetpack_publicize_feature_enabled` post metadata as True. It's a follow-up task that originated [here](https://github.com/Automattic/jetpack/pull/21423#discussion_r729675507). Props to @pablinos 

Fixes #

#### Changes proposed in this Pull Request:

* Publicize: enabled feature by default

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Create a new post
* Open the Jetpack sidebar
* Enable the RePublicize feature ([issue with instructions](https://github.com/Automattic/jetpack/pull/21366)
* Confirm that the toggle button is enabled, and it shows the "Sharing is enabled" message.
<img width="290" alt="Screen Shot 2021-10-15 at 8 38 02 AM" src="https://user-images.githubusercontent.com/77539/137480993-f83d1f72-3903-4345-b51f-983ae478aede.png">
